code/modules/hydroponics/beekeeping/beebox.dm

```diff
@@ -97,7 +97,7 @@
 
 		if(bees.len < get_max_bees())
 			var/freebee = FALSE //a freebee, geddit?, hahaha HAHAHAHA
-			if(bees.len <= 1) //there's always one set of worker bees, this isn't colony collapse disorder its 2d spessmen
+			if(bees.len <= 2) //there's always one set of worker bees + the queen, this isn't colony collapse disorder its 2d spessmen
```

Comment:
Why does this fix it.
A constructed apiary would have only the queen, making bees.len 1, which is covered by the ```<=```, so it produces 1 set of worker bees, this takes it to 2 bees total, the queen and 1 set of workers.

With your change, they'll surely get two free sets of workers now?

Reply:
hmm

Reply:
remie they aren't making ANY BEES AT ALL right now

Reply:
Did you test it? Also which plants did you use?

They worked recently

Reply:
Premade worked when I was testing #32523

Reply:
Goof probably didn't even have any plants around or something

Reply:
it's got nothing to do with plants.
I bet he didn't put any honeyframes in, meaning the hive couldn't support any bees, so ```if(bees.len < get_max_bees())``` would have been 1 < 0, which is false, so no bee generation at all.

Reply:
I had 3 fucking frames in tne box

Reply:
>premade

Did you test the cargo crate one?